exploratory_analysis.ipynb

```diff
@@ -440,199 +408,55 @@
-       "      <td>38.238397</td>\n",
-       "      <td>31231.301431</td>\n",
-       "      <td>332859.854344</td>\n",
-       "      <td>1248.706871</td>\n",
-       "      <td>1730.470081</td>\n",
-       "      <td>2057.923003</td>\n",
-       "      <td>2012.193230</td>\n",
-       "      <td>4013.007032</td>\n",
-       "      <td>2006.620956</td>\n",
-       "      <td>2005.811518</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>4</th>\n",
-       "      <td>CA</td>\n",
-       "      <td>134595.283254</td>\n",
-       "      <td>6.0</td>\n",
-       "      <td>57.368230</td>\n",
-       "      <td>38.436938</td>\n",
-       "      <td>48681.407228</td>\n",
-       "      <td>529973.319846</td>\n",
-       "      <td>973.046305</td>\n",
-       "      <td>1323.970289</td>\n",
-       "      <td>1574.302245</td>\n",
-       "      <td>2012.189748</td>\n",
-       "      <td>6057.368230</td>\n",
-       "      <td>2006.633301</td>\n",
-       "      <td>2005.863053</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "</div>"
-      ],
       "text/plain": [
-       "  state          index  state_code  county_code  measure_id     numerator  \\\n",
-       "0    AK  112525.615385         2.0   147.578307   29.536801   2264.548275   \n",
-       "1    AL  150852.201603         1.0    66.037915   38.263404   6434.582686   \n",
-       "2    AR  130280.253515         5.0    73.973545   38.413908   3640.168903   \n",
-       "3    AZ  132980.856540         4.0    13.007032   38.238397  31231.301431   \n",
-       "4    CA  134595.283254         6.0    57.368230   38.436938  48681.407228   \n",
-       "\n",
-       "     denominator    raw_value  confidence_interval_lower  \\\n",
-       "0   23845.850005  1548.142943                1658.959760   \n",
-       "1   60540.125959  1507.593773                2001.911959   \n",
-       "2   34124.412839  1451.283522                1861.930351   \n",
-       "3  332859.854344  1248.706871                1730.470081   \n",
-       "4  529973.319846   973.046305                1323.970289   \n",
-       "\n",
-       "   confidence_interval_upper  data_release_year    fips_code   start_year  \\\n",
-       "0                2589.626767        2012.097955  2147.578307  2006.638074   \n",
-       "1                2551.716180        2012.207914  1066.037915  2006.639719   \n",
-       "2                2558.896901        2012.220303  5073.973545  2006.666062   \n",
-       "3                2057.923003        2012.193230  4013.007032  2006.620956   \n",
-       "4                1574.302245        2012.189748  6057.368230  2006.633301   \n",
-       "\n",
-       "      end_year  \n",
-       "0  2005.535980  \n",
-       "1  2005.805485  \n",
-       "2  2005.805125  \n",
-       "3  2005.811518  \n",
-       "4  2005.863053  "
+       "2008.0    37633\n",
+       "2009.0    34590\n",
+       "2010.0    31600\n",
```

Comment:
Thanks for doing this year analysis. I vote for keeping things simple and picking a single year, and I think my preferred year to investigate is 2010 because 1) it has lots of data per your analysis here, 2) it is a census year so there is lots of other data for this year, 3) according to your analysis the distribution for raw_value is similar in 2009 and 2011 so there is a lot more data we could test with around 2010. What are your thoughts? I could see 2008 being another candidate, any others? 2012 is the most recent but doesn't appear to have much data so it is not a good option unfortunately.
I also think we should focus on a single measure, or maybe two at the most, and classify the raw values in to something like low, medium, high, etc. That will make it a classification problem at least to start, which I think is better than attempting to approach it as a regression problem from the start, but open to other ideas

Reply:
I agree that it is a good idea to keep things simple.  I didn't realize it is a census year, so it is a good idea to focus on 2010, which has a total of 31600 records.  

We can definitely expend the analysis later on.  Years 2009, 2010, 2011 have a total of 87990 rows.  The top measure 'Sexually transmitted infections' has 9384 records.

For the second group, years 2008,2007,2006,2005,2004,2003,2002,  there are a total of 168014 rows.  The top measure 'Premature Death' has a total of 21440 records.

For years 1997,1998,1999,2001,2000 there are a total of 168014 rows,14624 rows.  The top measure 'Premature Death' has a total of 14624 records.

So if we want to focus on Premature Death, we will have multi-year records.

I like the PR process as well.  Thanks for getting this going.